google/cloudfunctions_helpers.go

```diff
@@ -0,0 +1,97 @@
+package google
```

Comment:
I'd highly recommend taking a look at how we do this sort of thing for other resources- resource_kms_key_ring.go would be a good place to start.

Reply:
Removed this file